test/credits.js

```diff
@@ -0,0 +1,158 @@
+        await asserts.throws(credits.approveCreditTransaction(value, {from: alien}))
+    });
+
+    it('owner can approve not all credit request', async () => {
+        const borrower = accounts[3];
+        const amount = 1000;
+        let res = await credits.creditRequest(amount, {from: borrower});
+        res = await credits.creditRequest(amount, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount, {from: OWNER});
+        assert.equal(await credits.creditRequests(borrower), amount);
+        assert.equal(await credits.credits(borrower), amount);
+
+    });
+
+    it('borrower can return funds', async () => {
+        const borrower = accounts[3];
+        const amount = 1000;
+        await credits.creditRequest(amount, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount, {from: OWNER});
+        await  credits.returnFunds(borrower, amount, {from: OWNER});
+
+        assert.equal(await credits.credits(borrower), 0);
+
+    });
+
+
+    it('only owner can return funds', async () => {
+        const borrower = accounts[3];
+        const alien = accounts[2];
+        const amount = 1000;
+        await credits.creditRequest(amount, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount, {from: OWNER});
+        await  asserts.throws(credits.returnFunds(borrower, amount, {from: alien}));
+        assert.equal(await credits.credits(borrower), amount);
+
+    });
+
+    it('owner cannot return more funds than was borrowed', async () => {
+        const borrower = accounts[3];
+        const amount = 1000;
+        await credits.creditRequest(amount, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount, {from: OWNER});
+        await  asserts.throws(credits.returnFunds(borrower, amount+1, {from: OWNER}));
+
+        assert.equal(await credits.credits(borrower), amount);
+
+    });
+
+    it('should emit event after successful funds return ', async () => {
+        const borrower = accounts[3];
+        const amount = 1000;
+        await credits.creditRequest(amount, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount, {from: OWNER});
+        let result = await  credits.returnFunds(borrower, amount, {from: OWNER});
+        assert.equal(result.logs[0].event, 'FundsReturned');
+        assert.equal(result.logs[0].args.amount.valueOf(), amount);
+    });
+
+    it('owner can return half funds ', async () => {
+        const borrower = accounts[3];
+        const amount = 1000;
+        await credits.creditRequest(amount*2, {from: borrower});
+        await  credits.approveCreditTransaction(borrower, amount*2, {from: OWNER});
+        assert.equal(await credits.credits(borrower), amount*2);
+        await  credits.returnFunds(borrower, amount, {from: OWNER});
+        assert.equal(await credits.credits(borrower), amount);
+
+    })
+
```

Comment:
Tests should named with word: "should" :)
Like 'should allow owner to return half funds', etc.
Imagine you say this test should..